samples/visual_indicator/nodejs/.env

```diff
@@ -0,0 +1,3 @@
+MicrosoftAppId="<<MicrosoftAppId>>"
```

Comment:
Why your app folder name is viaual_indicator ?
If it is part of meeting, kindly follow that folder path structure.